input/pagecontent/export.md

```diff
@@ -577,7 +610,66 @@ Example response body:
   }
 ```
 
+Example manifest, `organizeOutputBy` kickoff parameter is `Patient`, and `allowPartialManifests` kickoff parameter is `true`:
+
+```json
+  {
+    "transactionTime": "2021-01-01T00:00:00Z",
+    "request" : "https://example.com/fhir/Patient/$export?_type=Patient,Observation",
+    "requiresAccessToken" : true,
+    "outputOrganizedBy": "Patient",
+    "output" : [{
+      "url" : "https://example.com/output/file_1.ndjson"
+    },{
+      "url" : "https://example.com/output/file_2.ndjson",
+      "continuesInFile": "https://example.com/output/file_3.ndjson"
+    },{
+      "url" : "https://example.com/output/file_3.ndjson"
+    }],
+    "deleted": [{
+      "type" : "Bundle",
+      "url" : "https://example.com/output/del_file_1.ndjson"      
+    }],
+    "error" : [{
+      "type" : "OperationOutcome",
+      "url" : "https://example.com/output/err_file_1.ndjson"
+    }],
+    "extension":{"https://example.com/extra-property": true},
+    "link": [{
+      "relation": "next",
+      "url": "https://example.com/output/manifest-2.json"
+    }]
+  }
+```
+
 ---
+#### Bulk Data Output File Organization
+
```

Comment:
Do we link to this section from the other places where organizeBy is defined as a param? I'm not seeing the links in source here

Reply:
Yup, it's linked from the parameter list using the header (href="#bulk-data-output-file-organization")